includes/Classifai/Features/ExcerptGeneration.php

```diff
@@ -45,6 +46,7 @@ public function __construct() {
 			ChatGPT::ID   => __( 'OpenAI ChatGPT', 'classifai' ),
 			GeminiAPI::ID => __( 'Google AI (Gemini API)', 'classifai' ),
 			OpenAI::ID    => __( 'Azure OpenAI', 'classifai' ),
+			ChromeAI::ID  => __( 'Chrome AI', 'classifai' ), // TODO: only add this Provider if the browser supports AI.
```

Comment:
Right now this Provider shows up even if your browser doesn't support Chrome AI. Would be great to remove it but doesn't cause any problems if you do select it. With the settings refactor in #502, I think will be much easier to remove this since all settings are registered in javascript at that point, so may leave this until it's integrated in there

Reply:
> Right now this Provider shows up even if your browser doesn't support Chrome AI. Would be great to remove it

Maybe keep showing it and display the steps of how to activate it (Like you provided in test steps)?

Reply:
@dkotter Updated as follows:  
- The Chrome AI option will be completely removed if `window.ai` is undefined (e.g., in browsers other than Chrome).  
- If built-in AI is not supported, "Not supported" will be displayed along with a link to documentation on how to enable AI in the browser.  
![Screenshot 2024-12-16 at 10 52 06 PM](https://github.com/user-attachments/assets/60f41828-a72b-4c82-aa16-29533d91af07)

- Built-in AI is supported.
![Screenshot 2024-12-16 at 10 52 30 PM](https://github.com/user-attachments/assets/e1a61361-8d9e-482e-9a3c-cc592cfa8119)

Please let me know if this looks good to you or if any updates are needed here.

Reply:
Looks great! Was actually working on something similar at the same time but this is better than what I had